internal/host/validator.go

```diff
@@ -1110,41 +938,41 @@ func comparisonBuilder(value float64) string {
 	return equals
 }
 
-func (v *validator) printSufficientNetworkLatencyRequirementForRole(c *validationContext, status ValidationStatus) string {
+func (v *validator) hasSufficientPacketLossRequirementForRole(c *validationContext) (ValidationStatus, string) {
+
+	var status ValidationStatus
+	var message string
+
+	if c.infraEnv != nil {
+		status = ValidationSuccessSuppressOutput
+	} else if len(c.cluster.Hosts) == 1 || c.clusterHostRequirements.Total.PacketLossPercentage == nil || common.GetEffectiveRole(c.host) == models.HostRoleAutoAssign || hostutil.IsDay2Host(c.host) {
+		// Single Node use case || no requirements defined || role is auto assign
+		status = ValidationSuccess
+	} else if len(c.host.Connectivity) == 0 {
+		status = ValidationPending
+	} else {
+		status, _, _ = v.validatePacketLossForRole(c.host, c.clusterHostRequirements, c.cluster.Hosts)
```

Comment:
I think you can avoid calling this twice (it's also called in the `ValidationFailure` case below) if you save off the packet loss and err values into variables here.